File: Java/Lists/StaticRoundedList/Listable.java
public interface Listable {
  void append(Object obj);
  void insert(int posicao, Object obj);
  Object selectOne(int posicao);
  void update(int posicao, Object novoObj);
  Object remove(int posicao);

  boolean isFull();
  boolean isEmpty();
  String print();
}

File: Kotlin/Lists/Listable.kt
interface Listable {
  fun append(): Any
  fun insert(posicao: Int, dado: Any)
  fun selectOne(posicao: Int): Any?
  fun update(posicao: Int, newDado: Any)
  fun remove(posicao: Int): Any?

  fun isFull(): Boolean
  fun isEmpty(): Boolean
  fun print(): String
}